### DISCUSSION TITLE
How to add feed after the feed.run()

### DISCUSSION THREAD

**Discussion by Vowenchu** [bmoscon/cryptofeed#1029] [Q&A] [answered]
Hi,

I am a new dev. I am currently stream option data from bybit. There are new symbols everyday as new option contracts created. 
I can run the script now but want to add new symbol in the add_feed everyday. How should this be done in cryptofeed?

**Comment by bmoscon** [comment, marked as answer]
you'd need to set something up to read the symbol data at whatever time/cadence you desire, and then create a new feed. There is an example that does this:

https://github.com/bmoscon/cryptofeed/blob/master/examples/demo_loop.py

You'd just need to make the appropriate calls to get the new symbol data from the exchange (https://github.com/bmoscon/cryptofeed/blob/master/cryptofeed/exchange.py#L78 make sure you set refresh to True when you make this call) and then use whatever new symbols to make new subscriptions.
